src/main/java/edu/ucsb/cs156/frontiers/services/OrganizationMemberService.java

```diff
@@ -62,4 +71,30 @@ public Iterable<OrgMember> getOrganizationMembers(Course course) throws NoSuchAl
         }));
         return orgMembers;
     }
+
+    public OrgStatus inviteOrganizationMember(RosterStudent student) throws NoSuchAlgorithmException, InvalidKeySpecException, JsonProcessingException {
+        Course course = student.getCourse();
+        return inviteMember(student.getGithubId(), course, "direct_member");
+    }
+
+    private OrgStatus inviteMember(int githubId, Course course, String role) throws JsonProcessingException, NoSuchAlgorithmException, InvalidKeySpecException {
+        String ENDPOINT = "https://api.github.com/orgs/" + course.getOrgName() + "/invitations";
+        HttpHeaders headers = new HttpHeaders();
+        String token = jwtService.getInstallationToken(course);
+        headers.add("Authorization", "Bearer " + token);
+        headers.add("Accept", "application/vnd.github+json");
+        headers.add("X-GitHub-Api-Version", "2022-11-28");
+        Map<String, Object> body = new HashMap<>();
+        body.put("invitee_id", githubId);
+        body.put("role", role);
+        String bodyAsJson = objectMapper.writeValueAsString(body);
+        HttpEntity<String> entity = new HttpEntity<>(bodyAsJson, headers);
+        try{
+            ResponseEntity<String> response = restTemplate.exchange(ENDPOINT, HttpMethod.POST, entity, String.class);
+        } catch (HttpClientErrorException e) {
```

Comment:
Maybe we should check the case where a student is already a member of the organization because they were added manually, and check to see whether we can distinguish this case in the error message from other kinds of errors that may have happened.

If we can, maybe we can set the status to MEMBER, for example, or OWNER, if we are able to determine that that is the correct status?

Or maybe we even do an API call first to check?